src/main/java/com/spotify/reaper/storage/PostgresStorage.java

```diff
@@ -496,11 +495,11 @@ public boolean updateRepairSchedule(RepairSchedule newRepairSchedule) {
     }
   }
 
-  private static UUID fromSequenceId(long insertedId) {
-    return new UUID(insertedId, UUIDs.timeBased().getLeastSignificantBits());
+  public static UUID fromSequenceId(long insertedId) {
```

Comment:
The problem with re-using these static methods is it creates a cyclic dependency between the `com.spotify.reaper.storage` and `com.spotify.reaper.storage.postgresql` packages.

If the two methods are moved something into the `com.spotify.reaper.storage.postgresql` package then the dependency direction remains only from `com.spotify.reaper.storage` to `com.spotify.reaper.storage.postgresql` as it is today.